ci/check-documentation.py

```diff
@@ -29,22 +30,41 @@
     'tt', 'u', 'ul', 'var',
 ])
 
-def find_md_files():
-    thirdparty_dir = OMZ_ROOT / 'demos' / 'thirdparty'
+thirdparty_dir = OMZ_ROOT / 'demos' / 'thirdparty'
+
 
+def find_md_files():
     for path in OMZ_ROOT.glob('**/*.md'):
         if thirdparty_dir in path.parents: continue
         yield path
 
 def main():
     all_passed = True
+    
+    md_check_cases = (f'{OMZ_ROOT}{os.sep}models{os.sep}intel{os.sep}index.md', 
+                     f'{OMZ_ROOT}{os.sep}models{os.sep}public{os.sep}index.md', 
+                     f'{OMZ_ROOT}{os.sep}demos{os.sep}README.md')
 
     def complain(message):
         nonlocal all_passed
         all_passed = False
         print(message, file=sys.stderr)
 
     for md_path in sorted(find_md_files()):
+        check_md_links = False
+        readme_files = list()
+
+        if os.path.normpath(md_path) in md_check_cases:
```

Comment:
This approach is a bit fragile, because if an index is moved/renamed, then the check will be silently skipped. I suggest that you move the index checks into an entirely separate loop over `md_check_cases`.

Reply:
You haven't really fixed this - your check will be silently skipped if an index file disappears.